scripts/Packages/Package_Gullwing__QFP_SOIC_SO/Readme.md

```diff
@@ -0,0 +1,151 @@
+# IPC gullwing generator
+
+This generator uses IPC-7351B equations and fillet definitions to derive a footprint for gullwing style packages. (fillet definitions can be changed by pointing to a different definition file.) The script generates footprints in zero orientation version A (pin 1 at top left corner).
+Pads are generated using rounded rectangle pads as suggested in preliminary releases of IPC-7351C.
+
+Examples of supported packages include: QFN, SOIC, SO.
```

Comment:
QFN is an example of a gullwing package? Is this a typo? Maybe examples could include `SOIC, TSSOP, HTSSOP, etc.` to capture more package types that are supported?

Reply:
typo, should have been qfp

Reply:
fixed